server/internal/capture/manager.go

```diff
@@ -34,13 +34,15 @@ func New(desktop types.DesktopManager, config *config.Capture) *CaptureManagerCt
 			return NewAudioPipeline(config.AudioCodec, config.AudioDevice, config.AudioPipeline, config.AudioBitrate)
 		}, "audio"),
 		video: streamSinkNew(config.VideoCodec, func() (string, error) {
-			return NewVideoPipeline(config.VideoCodec, config.Display, config.VideoPipeline, config.VideoMaxFPS, config.VideoBitrate, config.VideoHWEnc)
+			// use screen fps as default
+			fps := desktop.GetScreenSize().Rate
+			// if max fps is set, cap it to that value
+			if config.VideoMaxFPS > 0 && config.VideoMaxFPS < fps {
```

Comment:
I would prefer a `<=` here

Reply:
video max <= fps? why should it make a difference? if they are the same then it stays unchanged as well.

Reply:
Ah, my bad.
But there should be a check if `fps` is not 0 before setting it for the pipeline.

Reply:
True, theoretically xorg's modeline that reports 0 rate is valid but not gstreamers. I'd set it to 25 by default, beause that is also gstreamers default.